students/david_baylor/lesson3/Mail_Room_part1.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+"""
+Mail_room_part1.py
+By David Baylor on 12/3/19
+uses python 3
+
+Automates writing an email to thank people for their donations.
+"""
+
+def main():
+    data = [["bill", 100, 2, 50],["john",75, 3, 25]]
```

Comment:
you can define this inside `if __name__ == "__main__"` block and use it globally without having to pass it into all of the functions.